ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -10,11 +10,54 @@ import LlamaKit
 import Nimble
 import Quick
 import ReactiveCocoa
+import Foundation
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, #expectSentValue: T?, sentError expectSentError: E?, #complete: Bool) {
+	startSignalProducer(signalProducer, expectSentValues: expectSentValue.map { [$0] } ?? [], sentError: expectSentError, complete: complete)
+}
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, expectSentValues: [T] = [], sentError expectSentError: E?, #complete: Bool) {
```

Comment:
In previous RAC tests, there's never been a helper like this. I believe, but could be wrong, the preference is to have tests directly use the API they're testing. I say this partly to ask aloud.